index.html

```diff
@@ -456,32 +421,171 @@ <h2 id="constraints">
+                <code><a>CursorCaptureConstraint</a></code> it is capable of
+                supporting for this <a>display surface</a>.
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <p>When inherent properties of the underlying source of a user-selected
+        <a>display surface</a> change, for example in response to the end-user
+        resizing a captured window, and these changes render the capabilities
+        and/or settings of one or more constrainable properties outdated, the
+        user agent MUST queue a task to update all affected constrainable
+        properties at the same time.</p>
+        <section>
+          <h2>
+            Downscaling and frame decimation
+          </h2>
+          <p>For the purposes of the
+          <a href="https://www.w3.org/TR/mediacapture-streams#dfn-selectsettings">SelectSettings</a>
+          algorithm, the user agent SHOULD consider all possible combinations of
+          downscaled dimensions that preserve the aspect ratio of the original
+          <a>display surface</a>, and frame rates available through frame
+          decimation, as available
+          <a href="https://www.w3.org/TR/mediacapture-streams#dfn-selectsettings">settings dictionaries</a>.
+          </p>
+          <p>The intent is for the user agent to produce output that is close to
+          the ideal <code>width</code>, ideal <code>height</code>, and/or
+          ideal <code>frameRate</code> when these are specified, while at all
+          times preserving the aspect ratio of the original <a>display surface</a>.
+          </p>
+          <p>The user agent SHOULD downscale by
+          <a href="https://drafts.csswg.org/cssom-view/#dom-window-devicepixelratio">
+          <code>window.devicePixelRatio</code></a> by default, unless otherwise
+          directed by applied constraints.
+          </p>
+          <p>The user agent MUST NOT crop the captured output.</p>
+          <p>The user agent MUST NOT fire the
+          <a href="https://www.w3.org/TR/mediacapture-streams#event-mediastreamtrack-overconstrained">overconstrained</a> event or mute the track in any "overconstrained" situation.</p>
+          <div class="note">
+            <p>The max constraint type lets a web application provide a maximum
+            envelope for constrainable properties like width and height. This is
+            helpful to limit extreme aspect ratios, should the end-user resize a
+            <a>window</a> or <a>browser</a> surface while it is being captured.
+            </p>
+            <p>While min and exact constraints produce TypeError on
+            getDisplayMedia(), this specification does not alter the
+            track.applyConstraints() method. Therefore, they may instead produce
+            OverconstrainedError or succeed depending on values. Since there is
+            no effect of being "overconstrained", they serve no purpose.</p>
```

Comment:
Can exact behavior be specified on a per-constraint basis? Sounds like there is room for interpretation.
I find mentioning OverconstrainedError throwing followed by being "overconstrained" not having an effect/serving no purpose confusing. What are we trying to say?

Reply:
> Can exact behavior be specified on a per-constraint basis?

We're not doing so, so the only definitions of `exact` that are relevant, are those already defined in *mediacapture-main*. In all those cases *exact* is either identical to `min` = `max` = some value, or for constraints without `min` and `max` a single value that must be met, or fail with `OverconstrainedError`.

> I find mentioning OverconstrainedError throwing followed by being "overconstrained" not having an effect/serving no purpose confusing. What are we trying to say?

I'm probably saying too many things at once. I'm trying to say `min` and `exact` are not useful. What purpose does throwing `OverconstrainedError` serve here?

As to "overconstrained" having no effect, this refers to the language about "overconstrained" above this note.